samples/subsys/video/mega_full-featured/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  name: arducam mega camera full-featured test
+tests:
+  sample.video.test:
+    build_only: true
+    tags: video
+    platform_allow:
+      - rpi_pico
+    depends_on: video
```

Comment:
rpi_pico doesn't have any built-in video device so it's not setting "video" as one of the supported features 
```suggestion
```